src/System.Collections.NonGeneric/tests/CaseInsensitiveHashCodeProviderTests.cs

```diff
@@ -35,43 +36,41 @@ public static void Ctor_Empty_GetHashCodeCompare(object a, object b, bool expect
         [InlineData(5, 5, true)]
         [InlineData(10, 5, false)]
         [InlineData(5, 10, false)]
-        public static void Ctor_Empty_ChangeCurrentCulture_GetHashCodeCompare(object a, object b, bool expected)
+        public void Ctor_Empty_ChangeCurrentCulture_GetHashCodeCompare(object a, object b, bool expected)
         {
-            var cultureNames = new string[]
+            RemoteInvoke((ra, rb, rexpected) =>
             {
-                "cs-CZ","da-DK","de-DE","el-GR","en-US",
-                "es-ES","fi-FI","fr-FR","hu-HU","it-IT",
-                "ja-JP","ko-KR","nb-NO","nl-NL","pl-PL",
-                "pt-BR","pt-PT","ru-RU","sv-SE","tr-TR",
-                "zh-CN","zh-HK","zh-TW"
-            };
+                var cultureNames = new string[]
+{
+            "cs-CZ","da-DK","de-DE","el-GR","en-US",
+            "es-ES","fi-FI","fr-FR","hu-HU","it-IT",
+            "ja-JP","ko-KR","nb-NO","nl-NL","pl-PL",
+            "pt-BR","pt-PT","ru-RU","sv-SE","tr-TR",
+            "zh-CN","zh-HK","zh-TW"
+};
 
-            foreach (string cultureName in cultureNames)
-            {
-                CultureInfo newCulture;
-                try
-                {
-                    newCulture = new CultureInfo(cultureName);
-                }
-                catch (CultureNotFoundException)
-                {
-                    continue;
-                }
+                bool.TryParse(rexpected, out bool expectedResult);
 
-                CultureInfo origCulture = CultureInfo.CurrentCulture;
-                try
+                foreach (string cultureName in cultureNames)
```

Comment:
running it RemoteInvoke once per culture is over killing. I would say keep the changes and run it under the UWP and look if this will fail. If you can repro any issue I can take a closer look and get Windows fix it.